duckduckgo/Android
# Feature/david/refresh versions

Task/Issue URL: https://app.asana.com/0/488551667048375/1199900494889724

**Description**:
We want to improve the way we update dependencies in the project. With that in mind, we want to introduce https://jmfayard.github.io/refreshVersions/

The difference now is that all the library versions are stored in one file `versions.properties`, which the plugin uses to then indicate what version to use.

**Steps to test this PR**:
1. Pull the branch
2. Open `versions.properties`
3. Check that there are now comments with available updates
4. Execute `gradle refreshVersions`
5. Open `versions.properties`
6. Check all the available updates


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
